src/libs/ApiUtils.ts

```diff
@@ -38,12 +38,12 @@ function getApiRoot(request?: Request): string {
     const shouldUseSecure = request?.shouldUseSecure ?? false;
 
     if (shouldUseStagingServer) {
-        if (CONFIG.IS_USING_WEB_PROXY && !request?.shouldSkipWebProxy) {
```

Comment:
Not to go back and forth but `requiresShortLivedAuthToken` is unknown to the api library i.e. if we set that flag to true the request won't have the auth token added.

Reply:
That is true, are you saying that it may cause confusion making developers thing that a short lived auth token will be added when we pass `requiresShortLivedAuthToken: true`?

Reply:
Yes exactly. On the other hand if we use `shouldSkipWebProxy` and set that flag to true the request will actually skip the proxy

Reply:
What about `isForWebViewOrNewTab`? the main purpose of changing the name was to:

> That way we are not exposing the underlying "proxy" or "no proxy" to other areas of the code that don't really need to know about it.

`isForWebViewOrNewTab` doesn't cause confusion about the short lived auth token

Reply:
Yeah let's go with that